indico/modules/events/registration/forms.py

```diff
@@ -273,6 +273,14 @@ class TicketsForm(IndicoForm):
                                           description=_('Allow users to download their ticket from the registration '
                                                         'summary page.'))
 
+    ticket_template_id = SelectField(_('Ticket template'), [HiddenUnless('tickets_enabled', preserve_data=True),
+                                                            Optional()])
+
+    def __init__(self, *args, **kwargs):
+        super(TicketsForm, self).__init__(*args, **kwargs)
+        templates = kwargs.pop('templates')
```

Comment:
do this before the `super()` call. also, I would move the logic to get the list of templates in here (and pass the event to the form instead of a list of templates)